nutils/topology.py

```diff
@@ -508,7 +508,8 @@ def locate(self, geom, coords, *, tol, eps=0, maxiter=100, arguments=None, weigh
       coords = coords[...,_]
     if not geom.shape == coords.shape[1:] == (self.ndims,):
       raise Exception('invalid geometry or point shape for {}D topology'.format(self.ndims))
-    centroids = self.elem_mean(geom, geometry=geom, degree=2)
```

Comment:
There is another occurrence of `elem_mean` in mesh.py. Should we remove that one as well?